package.json

```diff
@@ -21,10 +36,13 @@
     "mocha": "^3.1.2",
     "mocha-lcov-reporter": "^1.2.0",
     "nsp": "^2.6.2",
-    "standard": "^8.1.0"
+    "optimize-js": "^1.0.2",
+    "standard": "^8.1.0",
+    "uglify-js": "^2.7.4",
+    "uglifyify": "^3.0.4"
   },
   "scripts": {
-    "build": "browserify lib/main.js -o browser/niceware.js",
+    "build": "browserify lib/main.js -g uglifyify | uglifyjs -cm | optimize-js > browser/niceware.js",
```

Comment:
could you add this as a separate build script, like `build-minify` or something? this may seem unconventional, but minification and other transforms sometimes introduce unexpected bugs [1], so i personally prefer to the least-processed js files available unless there is a noticeable performance issue.

[1] ex: https://diracdeltas.github.io/blog/backdooring-js/